modules/tinymce/src/core/test/ts/browser/newline/InsertNewLineTest.ts

```diff
@@ -162,4 +162,24 @@ describe('browser.tinymce.core.newline.InsertNewLineTest', () => {
     TinyAssertions.assertContent(editor, '<p><a href="#">a</a></p><p><a href="#"><img src="about:blank"></a></p>');
     TinyAssertions.assertSelection(editor, [ 1, 0 ], 0, [ 1, 0 ], 0);
   });
+
+  context('Blockquote manipulation', () => {
+    it('Press Enter in blockquote', () => {
```

Comment:
Should these be prefixed with the ticket ID I never really understood if that should be on all changes so that we can track it back to the ticket or just bug fixes.

Reply:
The It-blocks should, you're right. That was my oversight.